extensions/wikia/DesignSystem/services/DesignSystemGlobalFooterService.class.php

```diff
@@ -15,10 +15,13 @@ public function section() {
 		$this->response->setValues( [
 			'model' => $this->getVal( 'model' ),
 			'name' => $this->getVal( 'name' ),
-			'parentName' => $this->getVal( 'parentName' )
 		] );
 	}
 
+	public function image() {
+		$this->setVal( 'model', $this->getVal( 'model' ) );
+	}
```

Comment:
there's no need for a method here, you can render the template directly with renderPartial()